BookingQueueSubscriber/BookingQueueSubscriber.Services/VideoWeb/VideoWebServiceFake.cs

```diff
@@ -21,5 +23,11 @@ public Task PushParticipantsUpdatedMessage(Guid conferenceId, UpdateConferencePa
             PushParticipantsUpdatedMessageCount++;
             return Task.FromResult(HttpStatusCode.OK);
         }
+        
+        public Task PushAllocationToCsoUpdatedMessage(AllocationHearingsToCsoRequest request)
+        {
+            PushAllocationToCsoUpdatedMessageCount++;
```

Comment:
I can't see where this property PushAllocationToCsoUpdatedMessageCount is ever used, except for this increment. Is it needed?

Reply:
I just followed the same pattern of the previous counters  PushNewConferenceAddedMessageCount and PushParticipantsUpdatedMessageCount

Reply:
happy to remove them if it is needed

Reply:
Yeah, I think they're supposed to be used in unit tests. But it can be addressed when refactored.